api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -228,3 +254,54 @@ message Pass {
   // this pass.
   double uplink_center_frequency_hz = 10;
 }
+
+// A plan that specifies antenna configuration for tracking a satellite. A plan
+// is most often created from a pass but does not have to. A plan can contain
+// arbitrary antenna angles and communication frequencies.
+message Plan {
+  // The unique ID of the plan.
+  string id = 1;
+
+  // The id of the satellite to be tracked in the plan.
+  string satellite_id = 2;
+
+  // A description of the status of a plan.
+  enum Status {
+    // The plan is reserved for execution in the future at `aos_time`.
+    RESERVED = 0;
+
+    // The plan is currently executing and will end in the future at `los_time`.
+    EXECUTING = 1;
+
+    // The plan already finished executing successfully.
+    SUCCEEDED = 2;
+
+    // The plan failed to execute correctly.
+    FAILED = 3;
+
+    // The plan was canceled.
```

Comment:
maybe mention that a plan can be canceled for a reason that the satellite operator cannot control.

Reply:
Yeah it's good to add a comment now. We're thinking of adding a field later with the actual reason included too.